src/main/java/org/fundacionjala/pivotal/objects/CreateProject.java

```diff
@@ -0,0 +1,128 @@
+package org.fundacionjala.pivotal.objects;
+
+import org.fundacionjala.pivotal.common.Base;
+import org.fundacionjala.pivotal.common.CommonMethods;
+import org.fundacionjala.pivotal.common.Meth;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.ui.ExpectedConditions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.support.ui.ExpectedConditions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)